### PR TITLE
Fix Net current assets(liabilities) and Total less assets current liabilities showing as mandatory

### PR DESCRIPTION
**Pull request by caismith-ch**
Net current assets and Total less assets current liabilities are only mandatory when:
- CurrentAssets Total is not null
- Preprayments and accrued income is not null
- Creditors due within one year is not null

**Resolves**: SFA-904

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/api/accounts/validation/CurrentPeriodValidatorTest.java [hunk 1]
```diff
@@ -163,6 +163,39 @@ void validateBalanceSheetWithOtherLiabilitiesOrAssetsTotalNetAssetsError() {
                         ErrorType.VALIDATION.getType())));
     }
 
+    @Test
+    @DisplayName("ERROR - Mandatory elements missing for NetCurrentAssets and TotalAssetsLessCurrentLiabilities")
```
Comment (BSaunders95) [inline src/test/java/uk/gov/companieshouse/api/accounts/validation/CurrentPeriodValidatorTest.java]: Could we add tests for the scenario where either `CurrentAssets` or `OtherLiabilitiesOrAssets` is null while the other is not to improve unit test coverage?

**Comment by BSaunders95** [comment]
Looks good barring one small comment on code coverage. The alternative way to perform this validation would be to add a custom class-level annotation which has the added benefit of validating the model before it even hits the controller.
